pkg/runtime/validate.go

```diff
@@ -53,10 +57,17 @@ func (e *Executor) ValidateInputs() error {
 			return err
 		}
 
-		if _, err := os.Stat(e.filePath); err != nil {
+		if fi, err = os.Stat(e.filePath); err != nil {
 			zap.S().Errorf("file '%s' does not exist", e.filePath)
 			return errFileNotExists
 		}
+
+		mode = fi.Mode()
+		if !mode.IsRegular() {
```

Comment:
not sure if this is a good idea. IsRegular( ) returns a false if we pass a symlink.

@kanchwala-yusuf @williepaul @jlk WDYT?

Reply:
yes, it would return false for a symlink mode. I have added checks only for dir and regular files. Would user pass symlinks?

Reply:
could be an edgecase

Reply:
I think, we can tackle the edgecase if it gets reported